Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -953,11 +963,8 @@ static void WriteStage(ShaderCode& out, const pixel_shader_uid_data* uid_data, i
   if (!has_tex_coord)
     texcoord = 0;
 
-  if (stage.hasindstage)
   {
-    TevStageIndirect tevind;
-    tevind.hex = stage.tevind;
-
+    const TevStageIndirect tevind{.hex = stage.tevind};
```

Comment:
Designated initializers are a C++20 feature. GCC and Clang support it in C++17 but only as a nonstandard extension.

Reply:
It's also supported by MSVC for C++17.  I already used them in several places in #9497 (which has been merged, and would be a fair bit more awkward to adjust there because I use it to avoid variables), so I'd rather leave it like this.

Reply:
Hmm, fair. I guess we'll be able to use C++20 soon anyway, assuming macOS and Android don't hold us back...